src/main/java/org/ibp/api/rest/germplasmlist/GermplasmListResourceGroup.java

```diff
@@ -37,4 +41,29 @@
 		final List<TreeNode> children = this.germplamListService.getGermplasmListChildrenNodes(crop, programUUID, parentFolderId, onlyFolders);
 		return new ResponseEntity<>(children, HttpStatus.OK);
 	}
+
+	@ApiOperation(value = "Create a new Germplasm list")
+	@RequestMapping(value = "/crops/{crop}/germplasm-lists", method = RequestMethod.POST)
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'CROP_MANAGEMENT', 'GERMPLASM', 'MANAGE_GERMPLASM')")
+	@ResponseBody
+	public ResponseEntity<GermplasmListGeneratorDTO> create(
```

Comment:
Is there any reason why the service is responding with GermplasmListGeneratorDTO instead of GermplasmList? would it make more sense?

Reply:
Maybe, though in that case perhaps the parameter should also be a GermplasmListDTO, since it's a POST /germplasm-list.
Another option is POST /germplasm-lists/generation, in which case it makes more sense to respond with the generatorDTO.
In any case, it doesn't make much of a difference, it's just a way to answer to the client with the same object with the id filled in (and the entryNo if auto-generated)

@clarysabel